docs/getting-started/core-concepts.md

```diff
@@ -14,14 +14,15 @@ In order to access your UTXO, you will have to provide a valid mnemonic that wil
 
 Since the introduction of [deterministic wallet](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki), a Wallet is actually composed of multiple account. 
 
-For manipulation account, `Wallet.getAccount()` takes an optional (default: 0) account Id. 
+For manipulating multiple accounts, `Wallet.getAccount()` takes optional [options](/wallet/getAccount.md) where index allows either accessing or creating a specific account index. 
 
 ## Instantiation types
 
 A Wallet instance can be created from multiples types, which impact how much the Wallet can do. 
 In general, we expect you to initialize from a `mnemonic` or an `seed` (HD seed) or an `HDPrivateKey`, which allows wallet-lib to deal with HD Wallet (deterministic wallet).  
 
 In some other cases, you might want to instantiate Wallet from another input such as : 
-- `privateKey`: This allow to manage a single privateKey/publicKey set. Therefore, you will only have a single unique address to receive money. 
-- `HDPublicKey`: This allow a "watch-only" mode. You won't be able to spend anything, but this will allow you to track and monitor in real-time the addresses set of this public key. Use-case might be to always pay to another new 
+- `privateKey`: This allows to manage a single privateKey/publicKey set. Therefore, you will only have a single unique address to receive money. 
+- `HDPublicKey`: This allows a "watch-only" mode. You won't be able to spend anything, but this will allow you to track and monitor in real-time the addresses set of this public key. Use-case might be to always pay to another new 
```

Comment:
Looks like the second sentence was truncated? "...pay to another new..." address? wallet?
```suggestion
- `HDPublicKey`: This allows a "watch-only" mode. You won't be able to spend anything, but this will allow you to track and monitor in real-time the address set of this public key. Use-case might be to always pay to another new 
```

Reply:
@Alex-Werner I combined a comment and a suggestion so you probably missed the comment. It looks like there should be more to the second sentence ("Use-case might be to always pay to another new")?